flusher.go

```diff
@@ -114,9 +114,11 @@ type metricsSummary struct {
 	totalGlobalCounters int
 	totalGlobalGauges   int
 
-	totalLocalHistograms int
-	totalLocalSets       int
-	totalLocalTimers     int
+	totalLocalHistograms   int
+	totalLocalSets         int
+	totalLocalTimers       int
+	totalStatusChecks      int
+	totalLocalStatusChecks int
```

Comment:
Are there every cases for non-local status checks? I don't think there are, which means a chunk of this PR can be eliminated?